Dockerfile

```diff
@@ -20,8 +20,11 @@ WORKDIR /root/sources/install
 
 RUN chmod +x entrypoint.sh
 
+RUN chmod +x guacamole_install.sh
```

Comment:
shouldn't we move this to the package_base installation ? This way we don't have to update each dockerfile.